### PR TITLE
Fix: remove '?' when handling punctuation removal

### PR DESCRIPTION
**Pull request by chikaldirick**
This MR fixes an issue #25 by adjusting a regex used for punctuation removal (adding question mark `?` to the list of special characters).

### CODE REVIEW

File: src/components/TextForm.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default function TextForm(props) {
   };
 
   const handlePunctuation = () => {
-    let newText = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g, "");
+    let newText = text.replace(/[.,\/#!?$%\^&\*;:{}=\-_`~()]/g, "");
```
Comment (Dipti821) [inline src/components/TextForm.js:23]: Remove ampersand sign(&) as its a symbol not a punctuation mark.

**Review by Dipti821 — COMMENTED** [review]
Also make the changes to resolve the problems which you have mentioned in the issue.

File: src/components/TextForm.js (hunk 1 above)
Comment (chikaldirick) [reply src/components/TextForm.js]: Done within 563e612838c349acc5a1ed4e6729c778f569050b.

Comment (Dipti821) [reply src/components/TextForm.js]: > Done within [563e612](https://github.com/PCON-Hacktoberfest-2022/Textutils/commit/563e612838c349acc5a1ed4e6729c778f569050b).

Also correct the error of the brackets and duplicate functions which you mentioned.

Comment (chikaldirick) [reply src/components/TextForm.js]: Don't you think it'd be better to do in a separate PR?
I'd suggest pushing this forward and create a new one that won't bring any new stuff but just resolve existing issues.

Comment (Dipti821) [reply src/components/TextForm.js]: > Don't you think it'd be better to do in a separate PR? I'd suggest pushing this forward and create a new one that won't bring any new stuff but just resolve existing issues.

Okay raise the issue and solve it. Do it fast.

**Pull request merged**